server/cmwell-fts/src/main/scala/cmwell/fts/FTSService.scala

```diff
@@ -540,7 +545,22 @@ class FTSService(config: Config) extends NsSplitter{
               logger.error(s"exhausted all retries attempts. logging failures to RED_LOG and returning results ")
               promise.success(SuccessfulBulkIndexResult(indexRequests, bulkResponse))
             }
-          } else {
+          } else if(versionConflictErrors.length > 0){
+            val bulks = versionConflictErrors.map { sir =>
+              get(sir._1.getId, sir._1.getIndex)(executionContext).transform {
+                case Success(esInfotonRes) => esInfotonRes match {
+                  case Some(ti) => Success(SuccessfulIndexResult(sir._1.getId, Option(ti._1.indexTime)))
+                  case None => Success(SuccessfulIndexResult(sir._1.getId, None))
+                }
+                case Failure(e) => logger.error("Got exception from es while get index time for version conflict, error=" +
+                  e.getMessage + ",uuid=" + sir._1.getId)
+                  Failure(e)
+              }
+            }
+            val allSucessIndexes = cmwell.util.concurrent.successes(bulks.toList)
+            allSucessIndexes.map {x=> promise.success(bulkSuccessfulResult ++  SuccessfulBulkIndexResult(x))}
```

Comment:
`SuccessfulBulkIndexResult(x)` actually wraps the result with redundant `SuccessfulBulkIndexResult`

Reply:
changed the impl..removed it.so done